src/openrct2-ui/windows/Multiplayer.cpp

```diff
@@ -953,7 +951,7 @@ static void window_multiplayer_groups_paint(rct_window* w, rct_drawpixelinfo* dp
 
 static void window_multiplayer_groups_scrollpaint(rct_window* w, rct_drawpixelinfo* dpi, int32_t scrollIndex)
 {
-    ScreenCoordsXY screenCoords(0, 0);
+    auto screenCoords = ScreenCoordsXY{ 0, 0 };
```

Comment:
```suggestion
    ScreenCoordsXY screenCoords(0, 0);
```
There was no need to change it here, really, but since it's inoffensive, I'll keep

Reply:
I was only doing what you taught me, master :D

Reply:
I mean, I prefer the way you made it look, but the previous worked just fine and in the end they all evaluate to create a `ScreenCoordsXY` object initialized to `{ 0, 0}` :)